arelle/plugin/validate/NL/rules/fr_nl.py

```diff
@@ -62,14 +59,6 @@
     'font-size',
     'color'
 }
-XHTML_ALLOWED_TAGS = frozenset({
-    'b', 'br', 'div', 'em', 'i', 'li', 'ol', 'p',
-    'pre', 's', 'small', 'strong', 'sub', 'sup',
-    'table', 'td', 'th', 'tr', 'u', 'ul'})
-XHTML_ALLOWED_TYPES = {
-    'ol': frozenset({'1', 'a', 'A', 'i', 'I'}),
```

Comment:
> I don't think the allowed `<li>` types are covered by the schema, so it seems something like this should be kept, but testing the child elements rather than the parent.

Valid type values are defined [here](https://github.com/aaroncameron-wk/arelle-public/blob/b120ff22e0e126f3497cc137a072dcbf6d29b900/arelle/config/sbr-text-formatting.xsd#L358-L364). I've added an invalid type to the testcases.

> Pre-existing, you might also test `<ol><li type="&#x9;&#xA;&#xD;&#x20;a"/>`, which I believe is schema valid because type is xs:token, which is whiteSpace collapse.

You're correct, I've also added this to the testcases.

Reply:
> Valid type values are defined [here](https://github.com/aaroncameron-wk/arelle-public/blob/b120ff22e0e126f3497cc137a072dcbf6d29b900/arelle/config/sbr-text-formatting.xsd#L358-L364). I've added an invalid type to the testcases.

I meant that the schema comments say that `<ol><li type="circle">` and `<ul><li type="1">` are both invalid. The schema does not (cannot?) enforce that, so it still seems valuable to enforce in logic.